Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Represents the control that redistributes space between columns or rows of a Grid control.
+    /// </summary>
+    [TemplatePart(Name = SPLITTERRNAME, Type = typeof(Thumb))]
+    public partial class GridSplitter : Control
```

Comment:
Sorry about the Thumb control being sealed - I had overlooked that bit. I'm not sure it's really needed then. What's the reason for having any template parts? Shouldn't the class just be a ControlControl then?